modules/account-lib/package.json

```diff
@@ -36,11 +36,12 @@
   "dependencies": {
     "@bitgo/blake2b": "^3.0.1",
     "@bitgo/bls": "^0.1.0",
-    "@bitgo/statics": "^6.14.0",
+    "@bitgo/statics": "^6.15.0-rc.4",
```

Comment:
This is needed since contains the new statics config for Sol (the one it was reverted)